### PR TITLE
Refs [TASK][39471] Tạo giao diện xem danh sách tin tức tại trang chủ

### PR DESCRIPTION
**Pull request by okuloj**
## Related Tickets
- [#Task 39471: Tạo giao diện xem danh sách tin tức tại trang chủ](https://edu-redmine.sun-asterisk.vn/issues/39471)
## WHAT
- Giao diện xem danh sách tin tức tại trang chủ
## Evidence (Screenshot or Video)
Screenshot_1628670816

## Review Checklist

Category | View Point | Description | Self review | Reviewer2 (name)
--- | --- | --- | --- | ---
Conventions | Sun* coding conventions | https://github.com/framgia/coding-standards/tree/master/eng/android |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Conventions | Kotlin coding conventions | https://kotlinlang.org/docs/coding-conventions.html |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Sun* Redmine working process  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md |<li>- [ ] yes</li>|<li>- [ ] yes</li>  

## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/java/com/edu/news/screen/home/HomeFragment.kt [hunk 1]
```diff
@@ -2,13 +2,25 @@ package com.edu.news.screen.home
...
 import com.sun.news.R
+import kotlinx.android.synthetic.main.fragment_home.view.*
 
 class HomeFragment : BaseFragment() {
+    // Test tabLayout
+    private var listCategory =
+        listOf("Following", "Top-Headline",
+            "Sport", "Technology",
+            "Business", "Entertainment",
+            "Health")
```
Comment (luongth-2520) [inline app/src/main/java/com/edu/news/screen/home/HomeFragment.kt]: xoá comment, đưa string vào array strings

File: app/src/main/res/layout/fragment_home.xml [hunk 2]
```diff
@@ -5,4 +5,81 @@
...
+
+        <ImageView
+            android:layout_width="@dimen/dp_30"
+            android:layout_height="@dimen/dp_30"
+            android:src="@drawable/ic_baseline_search_24" />
+
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="Search"
```
Comment (luongth-2520) [inline app/src/main/res/layout/fragment_home.xml]: đưa vào strings

File: app/src/main/res/layout/fragment_home.xml [hunk 3]
```diff
@@ -5,4 +5,81 @@
...
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/linearLayoutSearch">
+
+        <TextView
+            android:id="@+id/textView"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginLeft="@dimen/dp_10"
+            android:text="Category"
```
Comment (luongth-2520) [inline app/src/main/res/layout/fragment_home.xml]: tương tự

File: app/src/main/res/layout/fragment_list_news.xml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    tools:context="com.edu.news.screen.home.HomeFragment">
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/recyclerViewListNews"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
```
Comment (luongth-2520) [inline app/src/main/res/layout/fragment_list_news.xml]: remove blank

**Review by luongth-2520 — APPROVED** [review]

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**